docs-site/content/27.0/api/search.md

```diff
@@ -776,6 +773,31 @@ curl -H "X-TYPESENSE-API-KEY: ${TYPESENSE_API_KEY}"  -X POST \
   </template>
 </Tabs>
 
+:::tip
+The `value` key in the preset configuration can also match the search parameters for <RouterLink :to="`/${$site.themeConfig.typesenseLatestVersion}/api/federated-multi-search.html`">Federated / Multi Search</RouterLink>. For example:
+
+```js
+  await client.presets().upsert("listing_view", {
+    value: {
+      searches: [
+        {
+          collection: "products",
+          q: "*",
+          sort_by: "popularity",
+        },
+        {
+          collection: "blog_posts",
+          q: "*",
+          sort_by: "published_at:desc",
+        }
+      ],
+    },
+  })
+```
+
+It's generally recommended to use single-search presets for flexibility. You can then combine them in a multi-search request using the `preset` parameter.
+:::
+
 You can use the preset configuration for a `GET .../search` end-point as well. 
 
 The only requirement is that for 
```

Comment:
@tharropoulos This part now conflicts with some information in the changes above. Could you edit this as needed? Or may be even delete it if not needed, with the new information above?

Reply:
Looks like GitHub is not rendering my comment against the right line in this PR view. 

My comment was about the following paragraph, not the one you deleted in https://github.com/typesense/typesense-website/commit/9ae2277f631808cadf19c712cec3718ea954ac89. (Let's add that back in).

![Screenshot 2024-09-19 at 3 50 42 PM](https://github.com/user-attachments/assets/e956b66f-3bd6-4152-8914-3c4b528eb67b)

Reply:
Is 235aaad resolving this?